jest.config.js

```diff
@@ -0,0 +1,191 @@
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
+
+module.exports = {
+  // All imported modules in your tests should be mocked automatically
+  // automock: false,
+
+  // Stop running tests after `n` failures
+  // bail: 0,
+
+  // Respect "browser" field in package.json when resolving modules
+  // browser: false,
+
+  // The directory where Jest should store its cached dependency information
+  // cacheDirectory: "C:\\Users\\Osivwi Okiti Eric\\AppData\\Local\\Temp\\jest",
+
+  // Automatically clear mock calls and instances between every test
+  clearMocks: true,
```

Comment:
🤔 Do we really need this config here for this task?

Reply:
Not sure about that. it will be removed.